content/en/guides/concepts/context-helpers.md

```diff
@@ -132,11 +132,11 @@ export default {
 }
 ```
 
-Want to use query parameters instead? You then use [context.query.id](/docs/2.x/internals-glossary/context#query) then.
+Want to use query parameters instead? You then use [context.query.id](/docs/2.x/internals-glossary/context#query).
 
 ### Redirecting users & accessing the store
 
-Accessing the Vuex store (when you have it set up through the `store` directory) is also possible through the context. It provides a `store` object which can be treated as `this.$store` in Vue components. In addition, we use the `redirect` method, an exposed helper through the context, to redirect the user in case the `authenticated` state is falsy.
+Accessing the [Vuex store](https://nuxtjs.org/docs/2.x/directory-structure/store) (when you have it set up through the `store` directory) is also possible through the context. It provides a `store` object which can be treated as `this.$store` in Vue components. In addition, we use the `redirect` method, an exposed helper through the context, to redirect the user in case the `authenticated` state is [falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy).
```

Comment:
```suggestion
Accessing the [Vuex store](/docs/2.x/directory-structure/store) (when you have it set up through the `store` directory) is also possible through the context. It provides a `store` object which can be treated as `this.$store` in Vue components. In addition, we use the `redirect` method, an exposed helper through the context, to redirect the user in case the `authenticated` state is [falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy).
```